src/assets/data/team.json

```diff
@@ -2,69 +2,69 @@
   {
     "img": "/src/assets/img/team/Benjamin Frost.webp",
     "name": "Benjamin Frost",
-    "lastRole": "Jury",
+    "lastRole": { "de": "Jury", "en": "Jury" },
     "hasMultipleRoles": true
   },
   {
     "img": "/src/assets/img/team/Jonas Wanke.webp",
     "name": "Jonas Wanke",
-    "lastRole": "Expert",
+    "lastRole": { "en": "Expert", "de": "Experte" },
```

Comment:
```suggestion
    "lastRole": { "en": "Expert", "de": "Bundestrainer" },
```

Reply:
I listened to you when you told me how to translate this, I liked the change and I still did it wrong